hack/create-dev-cluster.sh

```diff
@@ -55,6 +55,28 @@ if ! [ -n "$SSH_KEY_FILE" ]; then
 fi
 export AZURE_SSH_PUBLIC_KEY=$(cat "${SSH_KEY_FILE}.pub" | base64 | tr -d '\r\n')
 
+export AZURE_STANDARD_JSON_B64=$(echo '{
```

Comment:
Can you also update all the docs in docs/topics that have template examples?

Reply:
will do